http_parser.h

```diff
@@ -297,6 +344,7 @@ struct http_parser {
   unsigned int header_state : 7; /* enum header_state from http_parser.c */
   unsigned int index : 7;        /* index into current matcher */
   unsigned int lenient_http_headers : 1;
+  unsigned int traditional_case_http_headers : 1;
```

Comment:
I think this changes the size of struct http_parser, which would make it a semver-major change.